请问有paddlepaddle应用到Android的Demo吗

我看了那么久的文档，都不知道怎么使用，比如在手机上识别手写数字

@Xreki 我安装你给的这个[教程](https://github.com/PaddlePaddle/Mobile/blob/develop/benchmark/tool/C/README.md)在第二步的`make`时报错
`[ 17%] Linking CXX static library libpaddle_cuda.a`
`cpplint: Checking source code style`
`Traceback (most recent call last):`
`paddle/cuda/CMakeFiles/paddle_cuda.dir/build.make:146: recipe for target` `'paddle/cuda/libpaddle_cuda.a' failed`
`make[2]: *** [paddle/cuda/libpaddle_cuda.a] Error 1`
`make[2]: *** Deleting file 'paddle/cuda/libpaddle_cuda.a'`
`CMakeFiles/Makefile2:894: recipe for target 'paddle/cuda/CMakeFiles/paddle_cuda.dir/all' failed`
`make[1]: *** [paddle/cuda/CMakeFiles/paddle_cuda.dir/all] Error 2`
`Makefile:162: recipe for target 'all' failed`
`make: *** [all] Error 2`
不知道是不是我的go的版本问题，我的版本是`go version go1.6.2 linux/amd64`

--- a/benchmark/tool/C/README.md
+++ b/benchmark/tool/C/README.md
@@ -9,7 +9,7 @@ To compile and run this demo in an Android environment, please follow the follow
 
 1. Refer to [this document](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/mobile/cross_compiling_for_android_en.md) to compile the Android version of PaddlePaddle. After following the mentioned steps, make install will generate an output directory containing three subdirectories: include, lib, and third_party( `libpaddle_capi_shared.so` will be produced in the `lib` directory).
 
-2. Compile `inference.cc` to an executable program for the Android environment as follows:
+2. Compile `inference.cc` to an executable program for the Android environment as follows (run in `Mobile/benchmark/tool/C`):
     - For armeabi-v7a
     ```
     mkdir build

